src/inc/til/operators.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "rectangle.h"
+#include "size.h"
+#include "bitmap.h"
+
+#define _TIL_INLINEPREFIX __declspec(noinline) inline

```

Comment:
candidate for til.h itself

Reply:
Moved to `til.h` above all the includes of the sub-files so they can use it.